src/AbstractFormHandler.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Hostnet\Component\Form;
+
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * @author Yannick de Lange <ydelange@hostnet.nl>
+ */
+abstract class AbstractFormHandler implements NamedFormHandlerInterface
```

Comment:
An advantage of the current implementation would be, that we could slowly start transforming it into a data object we populate rather than making the handler stateful.